repose-aggregator/tests/performance-tests/test_vars/filters/scripting.yml

```diff
@@ -1,3 +1,8 @@
+# TODO: Remove this file? Don't want it to get picked up by the automated performance tests, but
+# TODO: keeping this file gives us a sensibly named way to run generic scripting tests.
+#
+# TODO: YAML does not seem to support include or inherit. As such, there is more duplication than
+# TODO: necessary because each scripting language only changes the "script_lang" variable.
```

Comment:
I guess I do see the case a little bit more. In theory (after we're done-done with the perf tests), you should only have to create a new performance test if you create a new filter (or write some logic that should be performance tested).

By deleting this file, we also have to add "or add a new scripting language" to the list. I don't think it's that big of a deal, though. Still good with deleting this file (which has already been done).